src/main/java/codeu/model/store/basic/HashtagStore.java

```diff
@@ -117,4 +117,16 @@ public Hashtag getHashtag(UUID id) {
     return null;
   }
 
+  public List<Hashtag> getHashtagsFromContent(String title) {
+    String[] hashtagList = title.split(" ");
+    List hashtagResults = new ArrayList();
```

Comment:
Should this be a list of Hashtags? I think you are returning a List of Strings.

Reply:
List without modifier is a list of objects, please use List Hashtag (forgot how to make angle brackets show up in github comments)